development/lapis_db.sql

```diff
@@ -0,0 +1,40 @@
+-- phpMyAdmin SQL Dump
+-- version 4.0.10deb1
+-- http://www.phpmyadmin.net
+--
+-- Host: localhost
+-- Generation Time: Jul 07, 2015 at 09:16 PM
+-- Server version: 5.5.41-0ubuntu0.14.04.1
+-- PHP Version: 5.5.9-1ubuntu4.9
+
+SET SQL_MODE = "NO_AUTO_VALUE_ON_ZERO";
+SET time_zone = "+00:00";
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8 */;
+
+--
+-- Database: `lapis_db`
+--
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `patterns`
+--
+
+CREATE TABLE IF NOT EXISTS `patterns` (
+  `id` int(255) NOT NULL AUTO_INCREMENT,
+  `author_id` int(255) NOT NULL,
+  `technology` longtext NOT NULL,
+  `description` longtext NOT NULL,
+  `code` longtext NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1 AUTO_INCREMENT=1 ;
```

Comment:
Yes, that is just the structure of the table I'm currently using exported from phpmyadmin so I can import it quickly to whatever DB instead of having to either re-create one or log into my sever and export it each time